buildconfig/CMake/CppCheck_Suppressions.txt.in

```diff
@@ -161,7 +161,7 @@ constParameterReference:${CMAKE_SOURCE_DIR}/Framework/Algorithms/src/AnnularRing
 returnByReference:${CMAKE_SOURCE_DIR}/Framework/API/inc/MantidAPI/FileProperty.h:90
 missingOverride:${CMAKE_SOURCE_DIR}/Framework/Algorithms/inc/MantidAlgorithms/BinaryOperation.h:59
 returnByReference:${CMAKE_SOURCE_DIR}/Framework/Kernel/inc/MantidKernel/EnumeratedString.h:62
-constParameterReference:${CMAKE_SOURCE_DIR}/Framework/Algorithms/src/AddSampleLog.cpp:345
+constParameterReference:${CMAKE_SOURCE_DIR}/Framework/Algorithms/src/AddSampleLog.cpp:344
```

Comment:
This is an easy cppcheck error to fix. Please do so and remove this line from the suppressions file.

Reply:
Removed suppression; cppcheck is green.